### PR TITLE
activity indicator accessibility label

### PR DESCRIPTION
**Pull request by harrieshin**
### Platforms Impacted
- [x] iOS
- [ ] macOS

### Description of changes

Add accessibility label to Fluent activity indicator that is similar to iOS native UIActivityIndicatorView.

### Verification

VO testing with spinner in search bar, cell with activity indicator.

### Pull request checklist

This PR has considered:
- [ ] Light and Dark appearances
- [x] VoiceOver and Keyboard Accessibility
- [ ] Internationalization and Right to Left layouts
- [ ] Different resolutions (1x, 2x, 3x)
- [ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui-apple/pull/280)

### CODE REVIEW

File: ios/FluentUI/Controls/ActivityIndicatorView.swift [hunk 1]
```diff
@@ -186,6 +187,25 @@ open class ActivityIndicatorView: UIView, ActivityViewAnimating {
         }
     }
 
+    private var useCustomAccessibilityLabel: Bool = false
+    open override var accessibilityLabel: String? {
```
Comment (Saadnajmi) [inline ios/FluentUI/Controls/ActivityIndicatorView.swift:191]: nit: newline between variable definitions. Also the ordering doesn't follow our swift guide https://microsoft.github.io/swift-guide/FileContentOrdering.html

File: ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -4,6 +4,9 @@
 //
 
 // Accessibility
+"Accessibility.ActivityIndicator.Animating.label" = "In progress";
+"Accessibility.ActivityIndicator.Stopped.label" = "Progress halted";
```
Comment (Saadnajmi) [inline ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings:8]: "Progress Halted" seems weird. Did we get that string from somewhere? What does normal UIActivityIndicator say?

**Review by Saadnajmi — CHANGES_REQUESTED** [review]

File: ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (harrieshin) [reply ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings:8]: yes it is from apple.

File: ios/FluentUI/Controls/ActivityIndicatorView.swift (hunk 1 above)
Comment (harrieshin) [reply ios/FluentUI/Controls/ActivityIndicatorView.swift:191]: usually i follow the order of the private and open... but i am conflicted on this one. the private var is only used for this public var so i kind of want them to be close

Comment (Saadnajmi) [reply ios/FluentUI/Controls/ActivityIndicatorView.swift:191]: I guess “A foolish consistency is the hobgoblin of little minds" is apt here

**Review by Saadnajmi — APPROVED** [review]

**Review by rdeassis — APPROVED** [review]

**Pull request merged**